.github/workflows/update.yml

```diff
@@ -0,0 +1,78 @@
+name: Update
+
+on:
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout l10n:main
+        uses: actions/checkout@v4
+        with: { ref: main }
+      - name: Refresh l10n:update
+        run: git push origin HEAD:update || true
+  branches:
+    needs: prepare
+    strategy:
+      max-parallel: 1
+      matrix:
+        ref: [master, beta, release, esr115]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout l10n:update
+        uses: actions/checkout@v4
+        with: { ref: update, path: l10n }
+      - name: Checkout firefox:${{ matrix.ref }}
+        uses: actions/checkout@v4
+        with:
+          repository: mozilla/gecko-dev
+          ref: ${{ matrix.ref }}
+          path: firefox
+          sparse-checkout: |
```

Comment:
This feels a bit brittle, as we risk missing new paths when they're added.

We should have a separate job that checks for new suspicious `locales` paths in gecko-dev and raises an error if that's not already included in a list of exceptions.

Reply:
Agreed, as a follow-up step. The expected `locales` paths can be deduced from the `l10n.toml` path configs, and a validation can check that each such path matches at least something.

Reply:
I've added #6 for this.